### ISSUE TITLE
toLocaleString() with UTC + offset

### ISSUE THREAD

**Issue by scleriot** [moment/luxon#201]
Hello,

I've got a date at UTC+3, and I'm trying to get the beginning of the week for the same timezone :
```js
const day = DateTime.local()
console.log(day.toLocaleString(DateTime.DATETIME_FULL)) //March 1 2018 at 14:24 UTC+3
console.log(console.log(day.startOf("week").toLocaleString(DateTime.DATETIME_FULL))) // February 26th 2018 at 03:00 UTC+3
```

Shouldn't the beginning of the week be computed for this timezone and output `February 26th 2018 at 00:00 UTC+3`?

**Comment by icambron** [comment]
A few things going on here. Bear with me.

First, note this is an issue formatting the time, not computing it. 

```js
> DateTime.local().startOf("week");
DateTime {
  ts: 2018-02-26T00:00:00.000+03:00,
  zone: UTC+3,
  locale: en-US }
```

So that's good; the problem is just in `toLocaleString`. The second piece is that what I see is this:

```js
> DateTime.local().startOf("week").toLocaleString(DateTime.DATETIME_FULL)
'February 26, 2018, 12:00 AM UTC'
```
That's different than what you have, but also a bug.

First, what's happening there? The formatter that ships with modern browsers doesn't know how to deal with "UTC+3" (it would be fine with an actual IANA zone like Europe/Minsk), so `toLocaleString()` can't really do it. Luxon has special code to handle that case by just formatting in UTC, but that code has a bug. It should have said it was 21:00 the day before, in UTC. So i need to fix that.

(For my future reference: This is more complicated than it looks -- if the format *doesn't* include a zone, we want to shift the time by the offset and render it as a sort of lying UTC. That's why the code does what it does. That change is [here](https://github.com/moment/luxon/commit/1ca6871a829f3ccb8a74e2cb030621ec9da1c64a). But it breaks if we actually show the string "UTC", which now makes the time wrong. So the shift-or-don't-shift decision has to be based on the format, which is complicating.)

In the long run, we'll fix this for real, once Chrome fixes a bug in its `Intl.DateTimeFormat` implementation.

But of course that's not what you saw anyway. Are you sure you're on the latest version? I've had to mess with that code a couple times.

**Comment by icambron** [comment]
This is "fixed" in 89af275, which will be released shortly. It's really only a partial fix, because you'll get the time formatted in UTC. It's the best we can do until https://bugs.chromium.org/p/chromium/issues/detail?id=364374 is fixed.

**Comment by durasj** [comment]
Hi @icambron, this should be now fixed thanks to  https://chromium.googlesource.com/v8/v8.git/+/198f1940aa7faf28d478facde091ad2eb4db5c59 from July 2020

[Testing this](https://festive-kirch-3ce62d.netlify.app/) on Chrome 88, I got correctly `Etc/GMT+3` for `(new Intl.DateTimeFormat(undefined, { timeZone: 'Etc/GMT+3'})).resolvedOptions().timeZone`. I can confirm it also works on Chrome 70, Firefox 84, Edge 87, Edge 18, Safari 13.1, latest iOS Safari and Samsung Internet using BrowserStack.

Maybe it's time to finally get rid of it? :slightly_smiling_face: If you don't have time to get to this soon and don't like removing workarounds as much as I do, I can submit the PR :slightly_smiling_face:
